jenkins-slaves/templates/jenkins-slave-ruby-template.yml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: jenkins-slave-ruby
+metadata:
+  annotations:
+    description: Jenkins slave for building ruby apps
+    iconClass: icon-ruby
+    tags: jenkins,ruby
+    version: 1.0.0
+  name: jenkins-slave-ruby
+objects:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    creationTimestamp: null
+    labels:
+      build: ${JENKINS_SLAVE_NAME}
+    name: ${JENKINS_SLAVE_NAME}
+  spec:
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: ${JENKINS_SLAVE_NAME}:latest
+    postCommit: {}
+    resources: {}
+    runPolicy: Serial
+    source:
+      contextDir: ${CONTEXT_DIR}
+      git:
+        ref: ${SOURCE_REPOSITORY_REF}
+        uri: ${SOURCE_REPOSITORY_URL}
+      type: Git
+    strategy:
+      dockerStrategy:
+        from:
+          kind: ImageStreamTag
+          name: jenkins-slave-base-centos7:latest
+      type: Docker
+    triggers:
+    - type: ConfigChange
+    - imageChange: {}
+      type: ImageChange
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    creationTimestamp: null
+    generation: 2
+    labels:
+      build: ${JENKINS_SLAVE_NAME}
+    name: jenkins-slave-base-centos7
```

Comment:
Do we want to add the RHEL version of the slave instead of centos?

Reply:
The reason that I chose the centos version of the image is that we're dealing with software collections packages for the build, which are not enabled by default on an OpenShift cluster.

Reply:
Thats fine with me. Just an inquiry